qemu/tests/driver_in_use.py

```diff
@@ -92,7 +92,8 @@ def run_subtest(sub_type):
     vm.verify_alive()
     error_context.context("Boot guest with %s device" % driver, logging.info)
```

Comment:
It would be confused for linux guest with rng device, as the driver_name is 'virtio' .